### DISCUSSION TITLE
2024/06/04/improving-upserts-ecto-postgresql/

### DISCUSSION THREAD

**Discussion by giscus** [katafrakt/website_comments#6] [Announcements]
# 2024/06/04/improving-upserts-ecto-postgresql/

With a little bit of a secret database sauce we managed to get the information about what our upsert did without resorting to tricks like comparing automatic timestamps.

https://katafrakt.me/2024/06/04/improving-upserts-ecto-postgresql/?utm_medium=email&utm_source=elixir-radar

**Comment by jwharrow** [comment]
Thanks for writing this. I think I'm missing something, but it looks like the inserted_at is updating despite the `:replace_all_except` flag in the example you posted.
